Purchases/Logging/Strings/AttributionStrings.swift

```diff
@@ -17,46 +17,118 @@ import Foundation
 // swiftlint:disable identifier_name
 enum AttributionStrings {
 
-    static let appsflyer_id_deprecated = "The parameter key rc_appsflyer_id is deprecated." +
```

Comment:
this file is probably easier to review side-by-side, since each line will be a change from `static let` to `case`